pages/index.tsx

```diff
@@ -5,7 +5,7 @@ const Index = () => {
   const router = useRouter()
 
   useEffect(() => {
-    router.push('/dao/MNGO')
+    router.push('/realms')
```

Comment:
should be configurable via ENV variable for deployment

Reply:
If i understand right we want to have realms page or eg /dao/mngo page as home page if i chose to have dao/mngo as my home page with the build, should users have access to realms page if they typed the url or should it be completely hidden from them ?

Reply:
Yeah, I think they should have access to the full ecosystem. It's very likely users would participate in many realms and this is why we are building this one stop shop for them.